src/lib/dialog/dialog-config.ts

```diff
@@ -69,5 +69,14 @@ export class MdDialogConfig {
   /** ID of the element that describes the dialog.  */
   ariaDescribedBy?: string | null = null;
 
+  /** Aria label to assign to the dialog element */
+  ariaLabel?: string | null = null;
+
+  /**
+   * Selector for an element to be focused when the dialog is opened. If omitted or the
+   * element is not found, the dialog will focus the first focusable element.
+   */
+  focusOnOpen?: string | null = null;
```

Comment:
Hmm I'm iffy about having two flags that do more or less the same. What about typing it something like `'root' | string | null`. That way it should show up in docs and IDEs that `root` is a special option.